packages/esm-appointments-app/src/appointments/common-components/appointments-actions.component.tsx

```diff
@@ -77,23 +69,10 @@ const AppointmentActions: React.FC<AppointmentActionsProps> = ({ appointment })
   };
 
   return (
-    <div style={{ display: 'flex', alignItems: 'center' }}>
-      {renderVisitStatus()}
-      {isFutureAppointment || (isTodayAppointment && (!handleCheckout || !hasActiveVisitToday)) ? (
-        <OverflowMenu aria-label="Actions" iconDescription={t('actions', 'Actions')} size="sm" flipped>
```

Comment:
- I don't know why we're checking for the absence of the `handleCheckout` function in this condition. It seems wrong to me. I've removed it from the conditional logic that determines whether to show the `OverflowMenu` or not.

Reply:
Wait... did you remove this OverflowMenu completely?

Reply:
No, I moved it to `AppointmentsTable`